pkg/virt-handler/vm.go

```diff
@@ -2710,6 +2787,13 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 		return err
 	}
 
+	if vmi.IsCPUDedicated() && vmi.Spec.Domain.CPU.IsolateEmulatorThread {
+		err = d.configureHousekeepingCgroup(vmi)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
Maybe I'm missing something, but what would cause the vm controller to run another sync loop for this vm to it could be un-paused? Should we re-enqueue this VM after a while to ensure that?

Reply:
@iholder-redhat i do not understand your comment. Can you please be more verbose?

Reply:
Nevermind, got my answer [here](https://github.com/kubevirt/kubevirt/pull/8233/files#r988726076). Sorry for not being clear.
You can ignore this comment